src/all/holonometria/src/eu/kanade/tachiyomi/extension/all/holonometria/Holonometria.kt

```diff
@@ -128,17 +129,17 @@ class Holonometria(
     }
 
     override fun pageListParse(document: Document): List<Page> {
-        return document.select("#js-mangaviewer img").mapIndexed { idx, img ->
+        return document.select(".manga-detail__swiper-slide.swiper-slide img").mapIndexed { idx, img ->
             Page(idx, "", img.attr("abs:src"))
```

Comment:
Prefer: 
```suggestion
            Page(idx, imageUrl = img.attr("abs:src"))
```

I am aware you did not touch this code.